great_expectations/render/renderer/inline_renderer.py

```diff
@@ -111,31 +116,33 @@ def _get_atomic_rendered_content_from_renderer_names(
                 logger.info(
                     f'Renderer "{renderer_name}" failed to render Expectation "{expectation_type}".'
                 )
-
-        if len(rendered_content) == 0 and isinstance(
-            render_object, ExpectationConfiguration
-        ):
-            logger.info(
-                f"""The following renderers failed to render Expectation "{expectation_type}":
-{non_default_prescriptive_renderer_names}
-Default renderer "{default_prescriptive_renderer_name}" will be used to render prescriptive content for ExpectationConfiguration.
-"""
-            )
-            renderer_rendered_content = (
-                InlineRenderer._get_renderer_atomic_rendered_content(
-                    render_object=render_object,
-                    renderer_name=default_prescriptive_renderer_name,
-                    expectation_type=expectation_type,
+                failed_renderer_name: FailedAtomicRendererName
+                if isinstance(render_object, ExpectationConfiguration):
+                    failed_renderer_name = FailedAtomicRendererName.PRESCRIPTIVE
+                    logger.info(
+                        f'Renderer "{failed_renderer_name}" will be used to render prescriptive content for ExpectationConfiguration.'
+                    )
+                else:
+                    failed_renderer_name = FailedAtomicRendererName.DIAGNOSTIC
+                    logger.info(
+                        f'Renderer "{failed_renderer_name}" will be used to render diagnostic content for ExpectationValidationResult.'
+                    )
+
+                renderer_rendered_content = (
+                    InlineRenderer._get_renderer_atomic_rendered_content(
```

Comment:
I realize this isn't caused by this PR but any way we can avoid accessing this private method?

Reply:
I'm not following the reasoning for this. This is used in two places and shouldn't be considered public by anyone using the `InlineRenderer`.

Reply:
@Kilo59 I agree that it should stay this way!